app/helpers/blacklight/catalog_helper_behavior.rb

```diff
@@ -172,12 +172,13 @@ def has_thumbnail? document
   ##
   # Render the thumbnail, if available, for a document and
   # link it to the document record.
-  # 
+  #
   # @param [SolrDocument]
   # @param [Hash] options to pass to the image tag
   # @param [Hash] url options to pass to #link_to_document
   # @return [String]
   def render_thumbnail_tag document, image_options = {}, url_options = {}
+    return "" unless has_thumbnail?(document) # do not render any html if there is no thumbnail
```

Comment:
sorry for all the other blank space diff noise, atom seems to remove those on save? not sure where it is happening